coq-htt.opam

```diff
@@ -2,17 +2,18 @@ opam-version: "2.0"
 version: "dev"
 
 maintainer: [
-  "Ilya Sergey <ilya.sergey@yale-nus.edu.sg>"
-  "Yasunari Watanabe <yasunari@comp.nus.edu.sg>"
+  "Aleks Nanevski"
+  "Alex Gryzlov"
+  "Germán Andrés Delbianco"
```

Comment:
```suggestion
  "Aleks Nanevski <aleks.nanevski@imdea.org>"
  "Alex Gryzlov <alex.gryzlov@imdea.org>"
  "Germán Andrés Delbianco <german@nomadic-labs.com>"
```

Reply:
> "Alex Gryzlov <alex.gryzlov@imdea.org>"
The right email is: aliaksandr.hryzlou@imdea.org.

Reply:
My bad

Reply:
@ilyasergey can you apply the patch?